docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+- `blockSignPublicKey`: is the address that will sign the block when forged. It can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used;
+- `vrfPubKey`: this is the key created in the previous step;
+- `value`: the amount of ZEN expressed in Satoshis (10^8) to put on stake, that will represent your voting power.
+
+#### Create a Forging Stake using Web3js
+Here is an example (written in JavaScript to be run in the Remix IDE) of creating the forging stake with a smart contract interaction:
+```
+(async () => {
+    try {
+
+        console.log('Running testWeb3 - Delegate or create a new Forging Stake script...')
+
+        const smartContractAddress = "0x0000000000000000000022222222222222222222";
+        const amount = 0.001; // amount of ZEN of the new stake
+        const accounts = await web3.eth.getAccounts();
+        console.log('Account ' + accounts[0]);
+
+        const abi = require("browser/contracts/artifacts/abi_fs.json"); // Change this for different path
+
+        const contract = new web3.eth.Contract(abi, smartContractAddress, {from: accounts[0]});
+
+        // Testing method:  delegate
+        const ownerAddress = accounts[0];
+        // pick one of the existing forgers, or create a new forger
+        const blockSignPublicKey = "0x" + YOUR_BLOCK_SIGN; // the address that will sign the block when forged; it can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used
+        const forgerVrfPublicKey = "0x" + YOUR_VRF_KEY; // this is the key created in the previous step
+        const first32BytesForgerVrfPublicKey = forgerVrfPublicKey.substring(0, 66);
+        const lastByteForgerVrfPublicKey = "0x" + forgerVrfPublicKey.substring(66, 68);
+
+        methodName = 'delegate';
+        console.log('Response for '+ methodName);
+
+        const response = await contract.methods.delegate(blockSignPublicKey,first32BytesForgerVrfPublicKey,lastByteForgerVrfPublicKey,ownerAddress).send({value: amount *10**18}).then(console.log);
+
+    } catch (e) {
+        console.log("Error:" + e.message);
+    }
+  })()
+```
+⚠️ **Remember to replace YOUR_BLOCK_SIGN and YOUR_VRF_KEY with the actual keys, and to specify the path of the native contract ABI. The contract ABI is not provided here, but it is available with the complete Remix workspace.**  
+⚠️ **Notice that this operation involves increasing the voting power of the forger defined by the blockSignPublicKey and forgerVrfKey, and with that the chance to produce blocks. The operation is a delegation of stake where no rewards will be automatically forwarded to you if you are not the owner of both blockSignPublicKey and forgerVrfKey. Ensure ownership before executing the transaction.
```

Comment:
Please clarify this a bit, per my understanding the node that has access to the blockSign and forgerVrf keys and produces the block will always pay rewards to itself to the first wallet account key in the local node's wallet.

Reply:
Yes, maybe it's not perfectly clear then when it says `The operation is a delegation of stake where...` that it only refers when the blocksignPublicKey is the one of a third forger and not the current node (describing the delegation process). I'll rephrase the sentence